### ISSUE TITLE
"terminated by other long poll or webhook (409)"

### ISSUE THREAD

**Issue by persian-telegram** [python-telegram-bot/python-telegram-bot#582]
### Steps to reproduce
1. running the code in console

2. This is a very weird error. It happens between 20-50 hours of using one bot and then i have to make another bot and move the exact same code to the new bot

3.

### Expected behaviour
Tell us what should happen
works normal

### Actual behaviour
Tell us what happens instead
gives warning/errors in the console
### Configuration
**Operating System:**
ubuntu 14.04

**Version of Python, python-telegram-bot & dependencies:**

``$ python -m telegram``

### Logs
Insert logs here (if necessary)
ERROR:telegram.ext.updater:Error while getting Updates: Conflict: terminated by other long poll or webhook (409)
WARNING:__main__:Update "None" caused error "Conflict: terminated by other long poll or webhook (409)"

**Comment by jh0ker** [comment]
Usually, this will happen if you run two instances of your bot at the same time. By the way, have you considered joining our telegram group at https://t.me/pythontelegrambotgroup You will get help quicker, and it is easier for us to respond.

**Comment by tsnoam** [comment]
As @jh0ker mentioned this is something that happens when you try to run to webhook bots in parallel.
Closing this issue for now.
If you feel like opening a new issue, please fill the information requested in the new issue template.
